docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -1,120 +1,136 @@
+  
+  Including:
 
--   `volumeHandle`
--   `volumeAttributes` (optional)
-
-    Including:
-
-    -   `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
-    -   `storage_type`: <SVC | A9K | DS8K>
-    -   `volume_name`: _Volume name_
-    -   `array_address`: _Array address_
+    - `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
+    - `storage_type`: <`SVC` | `A9000` | `DS8K`>
+    - `volume_name`: _Volume name_
+    - `array_address`: _Array address_
 
 To find the `volumeHandle`, use one of the following procedures:
 
-- Through command line (for Spectrum Virtualize Family):
+- **For Spectrum Virtualize Family**
 
-  `lsvdisk <volume name> | grep vdisk_UID`
-    
-  ```
-  lsvdisk vol0 | grep vdisk_UID
-  vdisk_UID 600507640082000B08000000000004FF
-  ```
+  The `volumeHandle` is formatted as `SVC:id;vdisk_UID`.
 
-- Through command line (for FlashSystem A9000 and A9000R):
+  - Through command line:
+    Find both the `id` and `vdisk_UID` attributes, by using the `lsvdisk` command.
 
-  `vol_list_extended vol=<volume_name>`
+    For more information, see **Command-line interface** > **Volume commands** > **lsvdisk** within your specific product documentation on [IBM Docs](https://www.ibm.com/docs/en).
 
-  For example, for `vol1`:
+  - Through the management GUI:
 
-  ```
-  A9000>> vol_list_extended vol=vol1
-  Name   WWN                                Product Serial Number     
-  vol1   6001738CFC9035E8000000000091F0C0   60035E8000000000091F0C0 
-  ```
+    1. Select **Volumes** > **Volumes** from the side bar.
 
-- Through the Spectrum Virtualize management GUI:
+        The **Volumes** page is displayed.
 
-  1.  Select **Volumes** > **Volumes** from the side bar.
+    2. Browse to the volume that the port is on and right-click > **Properties**.
 
-      The **Volumes** page appears.
+      The Properties window is displayed. Use the **Volume ID** and **Volume UID** values.
 
-  2.  Browse to the volume that the port is on and right-click > **Properties**.
+    For more information about Spectrum Virtualize products, find your product information in [IBM Documentation](https://www.ibm.com/docs/).
+  
+- **For FlashSystem A9000 and A9000R:**
 
-      The Properties window appears. Use the UID number.
+  The `volumeHandle` is formatted as `A9000:id;WWN`.
+  
+  - Through command line:
 
-      For more information about Spectrum Virtualize products, find your product information in [IBM Documentation](https://www.ibm.com/docs/).
+    Find the `id` and `WWN` for the volume, by using the `vol_list -f` command.
 
-- Through the IBM Hyper-Scale Manager user interface for FlashSystem A9000 and A90000R storage systems:
+    For more information, see **Reference** > **Command-line reference (12.3.2.x)** > **Volume management commands** > **Listing volumes** within your specific product documentation on [IBM Docs](https://www.ibm.com/docs/en).
```

Comment:
I thought we don't specify the user's storage version

Reply:
12.3.2.a an higher are what are currently supported. in addition, this is what the title is called in the documentation and so that is what it is referring to (12.3.2.x).